src/style/stars.css

```diff
@@ -0,0 +1,25 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  margin-left: 4px;
+  background-position: center;
+  background-image: url("../images/star.svg");
```

Comment:
it isn`t so necessary, but you can also make your code shorter and write:
background-image: url("../images/star.svg") no-repeat center;